first-analysis-steps/lhcb-dev.md

```diff
@@ -74,13 +74,13 @@ If you want to make changes to a software package, you will need to set up a
 development environment. `lb-dev` is your friend here:
 
 ```bash
-lb-dev --name DaVinciDev DaVinci/v45r1
+lb-dev --name DaVinciDev DaVinci/v45r5
```

Comment:
There is some discussion in #242 about bumping the version. As far as I could tell it's OK in the first-steps lessons, but we should change it coherently.